skt/templates/partials/test_failure.j2

```diff
@@ -2,14 +2,18 @@ One or more kernel tests failed:
 
 {% for job in report_jobs %}
   {% set kernel_arch = job.cross_compiler_prefix.split('-')[0] if job.cross_compiler_prefix else job.kernel_arch %}
-  {# If the job has no test results, then it passed. #}
-  {% if job.test_results %}
-  {{ kernel_arch }}: FAILED
+  {# Check the failed_tasks for each recipe to figure out if anything failed #}
+  {% set result = namespace(res='PASSED') %}
+  {% for test_result in job.test_results %}
+    {% if test_result.failed_tasks %}
+      {% set result.res = 'FAILED'  %}
+    {% endif %}
+  {% endfor%}
+  {{ kernel_arch }}: {{ result.res }}
+  {% if result == 'FAILED' %}
```

Comment:
I know little to nothing about Jinja. Is it OK that we assign to `result.res`, but then check just `result`?

Reply:
Good comment, ha! I was changing the previous version to the namespacing because jinja takes variable scope really seriously and didn't notice the missing jobs since I was looking at the limited template. Fixed now